src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java

```diff
@@ -26,60 +26,42 @@
     public static final String AS_RAW_RMS_MAPPING_QUALITY_KEY =     "AS_RAW_MQ";
     public static final String AS_CULPRIT_KEY =                     "AS_culprit";
     public static final String AS_VQS_LOD_KEY =                     "AS_VQSLOD";
-    public static final String ALLELE_BALANCE_HET_KEY =             "ABHet";
-    public static final String ALLELE_BALANCE_HOM_KEY =             "ABHom";
```

Comment:
These would be useful, but there's no code to generate them.  You can delete them, but they're not bad annotations.